Workflow/Sources/WorkflowHost.swift

```diff
@@ -115,3 +123,20 @@ public final class WorkflowHost<WorkflowType: Workflow> {
         outputEvent
     }
 }
+
+// MARK: - HostContext
+
+/// A context object to expose certain root-level information to each node
+/// in the Workflow tree.
+final class HostContext {
```

Comment:
class?

Reply:
sorry, are you asking why is it a class? assuming 'yes', then it's because i think we will eventually want to allow different parts of the tree to be able to mutate some of its properties and have those changes be visible elsewhere. e.g. you're handling an action and want to remember that 'some node in the tree was invalidated' by the time the root/host has to do the next render.